packages/http-client/src/remote-admin-api.ts

```diff
@@ -67,17 +93,26 @@ export class RemoteAdminApi implements AdminApi {
     })
   }
 
-  async startIndexingModels(modelsIDs: Array<StreamID>): Promise<void> {
+  async startIndexingModels(
+    modelIDs: Array<StreamID>,
+    indices?: Array<ModelFieldsIndex>
+  ): Promise<void> {
     const code = await this.generateCode()
     await this._fetchJson(this.getModelsUrl(), {
       method: 'post',
       body: {
-        jws: await this.buildJWS(this._getDidFn(), code, this.getModelsUrl().pathname, modelsIDs),
+        jws: await this.buildModelsJWS(
+          this._getDidFn(),
+          code,
+          this.getModelsUrl().pathname,
+          modelIDs,
+          indices
```

Comment:
Looks like you need to pass the body as an object. Also, convert the models array to strings.

```suggestion
          {
            models: modelsIDs.map((streamID) => streamID.toString()),
            indices
          }
```